File: ProgramasPython/URI.Problems/problem1065.py
## 
## Pares entre Cinco Numeros
## URI ONLINE - Problema 1065 - Iniciante
##
## Created by Jon on 16.03.2016
##	

# -*- coding: utf-8 -*-

cont = 0

for x in range(1,6):
	valor = input()
	if not(valor%2):
		cont+=1
		
print ("%d valores pares"%(cont))

File: Programas_C/URI/problem1065.c
#include <stdio.h>

int main(){
	int cont = 0, i = 5, valor;

	while(i--){
		scanf("%i", &valor);
		if (valor % 2 == 0){
			cont++;
		}
	} 

	printf("%i valores pares\n", cont);

	return 0;
}